client-spark/spark3/src/main/java/org/apache/spark/shuffle/RssShuffleManager.java

```diff
@@ -220,6 +222,17 @@ public RssShuffleManager(SparkConf conf, boolean isDriver) {
     }
   }
 
+  @VisibleForTesting
+  protected static ShuffleDataDistributionType getDataDistributionType(SparkConf sparkConf) {
+    RssConf rssConf = RssSparkConfig.toRssConf(sparkConf);
+    if ((boolean) sparkConf.get(SQLConf.ADAPTIVE_EXECUTION_ENABLED())
```

Comment:
`sparkConf.get(SQLConf.ADAPTIVE_EXECUTION_ENABLED()` should be used to be consistent with spark, because different spark versions have different default value

Reply:
cc @jerqi